.travis.yml

```diff
@@ -19,9 +19,9 @@ before_install:
   - sh docker/run-opendj.sh start
   
 script:
-  - SBT_OPTS=-Xmx3g sbt clean coverage test -Denv.type=test -Dmysql.host=localhost -Dmysql.port=3307 -Ddirectory.url=ldap://localhost:3389 -Ddirectory.password=testtesttest it:compile
+  - SBT_OPTS=-J-Xmx3g sbt clean coverage test -Denv.type=test -Dmysql.host=localhost -Dmysql.port=3307 -Ddirectory.url=ldap://localhost:3389 -Ddirectory.password=testtesttest it:compile
 
 after_success:
-  - SBT_OPTS=-Xmx3g sbt coveralls
+  - SBT_OPTS=-J-Xmx3g sbt coveralls
```

Comment:
does the `-J` change mean you have to change the docker build / run things too?

Reply:
possibly also the docker-compose in fc-develop

Reply:
things are a bit weird here. I don't know the case for rawls yet. but in sam, I didn't find where the version for docker is defined, and it's still 0.13, which is why I can't add `-J` for docker file